src/main/jbake/content/community/docs/developer-guide/alerts-v2.adoc

```diff
@@ -872,6 +872,253 @@ Url can be a list of valid link:https://hc.apache.org/httpcomponents-core-ga/htt
+| POST | /hawkular/alerter/notifications | link:#++POST_P8S_NOTIFICATIONS++[Listen Prometheus Notifications.]
+|===
+
+==============================================
+
+[[GET_P8S_STATUS]]
+*Endpoint GET `/awkular/alerter/status`*
+
+Ping status of the Prometheus Alerter +
+
+*Response*
+
+*Status codes*
+[cols="^20,55,^25", options="header"]
+|=======================
+|Status Code|Reason|Response Model
+
+|200|Prometheus Alerter is up and running|Success
+
+|=======================
+
+==============================================
+
+
+==============================================
+
+[[POST_P8S_NOTIFICATIONS]]
+*Endpoint POST `/awkular/alerter/notifications`*
+
+Receive a Prometheus Notification and store it as a Hawkular Event. Expects a format like: +
+
+```
+{
+  "receiver":"...",
+  "status":"...",
+  "alerts":[̣
+    {
+      "status":"...",
+      "labels":{},
+      "annotations":{},
+      "startsAt":"...",
+      "endsAt":"...",
+      "generatorURL":"..."
+    },
+    {...},
+    {...}
+  ],
+  "groupLabels":{},
+  "commonAnnotations":{},
+  "externalUrl":"...",
+  "version":"...",
+  "groupKey":"..."
+}
+```
+
+A Prometheus Notification is mapped into a Hawkular Event with the following convention: +
+ +
+
+[cols="15,15", options="header"]
+|=======================
+|Prometheus Notification|Hawkular Event
+
+|notification.groupLabels.tenant|event.tenantId
+|notification.groupLabels.dataId|event.dataId
+|"prometheus"|event.category
+|notification.commonAnnotations.description|event.text
+|notification.commonAnnotations.summary|event.text
+|notification.groupLabels.alertName|event.text
+|notification.groupLabels.alertName|event.context.alertName
+|notification|event.context.json
+
+|=======================
+
+
+ notification.groupLabels.dataId -> event.dataId
+
+
+*Body*
+
+[cols="15,35,^15", options="header"]
+|=======================
+|Required|Description|Data Type
+
+|Yes|Prometheus Notification|Object
+
+|=======================
+
+
+
+*Response*
+
+*Status codes*
+[cols="^20,55,^25", options="header"]
+|=======================
+|Status Code|Reason|Response Model
+
+|200|Successfully processed|Success
+|400|Bad Request/Invalid Parameters|<<ApiError>>
+
+|=======================
+
+
+
+==============================================
+
+== Apache Kafka Integration
+
+=== Apache Kakfa Alerter
+
+The Kafka Alerter will listen for triggers tagged with "Kafka" tag. The Alerter will create a consumer to a Kafka Topic
+with the config provider into the Trigger context. The Alerter will convert Kafka records into Hawkular Data or Events
+and send them into the Alerting engine.
+
+The Kafka Alerter uses the following conventions for trigger tags and context:
+
+[cols="^2,10"]
+|=======================
+| *Required* |
+_trigger.tags["Kafka"] = "<value reserved for future uses>"_ +
+ +
+ An "Kafka" tag is required for the alerter to detect this trigger will listen to a Kafka topic. +
```

Comment:
"The "Kafka" tag is required for the alerter to detect this trigger and listen to the Kafka topic."